autoware.repos

```diff
@@ -71,18 +75,22 @@ repositories:
   sensor_kit/sample_sensor_kit_launch:
     type: git
     url: https://github.com/autowarefoundation/sample_sensor_kit_launch.git
-    version: 4f949b50a0f45a29161f3a56578cfd0ef0056b2c
+    version: 3de568d173547c3605ef444944c540ad49db7471
+  sensor_kit/awsim_sensor_kit_launch:
```

Comment:
I think this should be added to the `main` branch first.

Reply:
Waiting for https://github.com/autowarefoundation/autoware/pull/2912.

Reply:
Merged!

Reply:
I'll rebase this branch to make it simple. And then merge it.

Reply:
(Revert 2e7cc865ca23ce912885f18c1d64d59d11399926 and rebase main?)